quodlibet/po/fr.po

```diff
@@ -523,9 +513,8 @@ msgid "Internet Radio"
 msgstr "Radio Internet"
 
 #: ../quodlibet/browsers/iradio.py:252
-#, fuzzy
 msgid "Downloading station list"
-msgstr "Télécharger les fichiers"
+msgstr "Télécharger la liste de station"
```

Comment:
I think English is different though here - we use singular forms for the first noun in compound nouns typically (_car parks_, _boat races_, _meeting notes_ etc)